homeassistant/components/lifx/light.py

```diff
@@ -448,9 +448,33 @@ def __init__(self, bulb, effects_conductor):
         """Initialize the light."""
         self.bulb = bulb
         self.effects_conductor = effects_conductor
-        self.registered = True
+        self._attr_available = True
         self.postponed_update = None
         self.lock = asyncio.Lock()
+        self._attr_unique_id = self.bulb.mac_addr
+        self._attr_name = self.bulb.label
+        """set the coldest/warmest color_temp that this light supports."""
+        kelvin = lifx_features(self.bulb)["max_kelvin"]
```

Comment:
`lifx_features()` is called multiple times now in this constructor, maybe store the result once and re-use it?

Reply:
Done.